src/controllers/playback/video/index.js

```diff
@@ -1237,7 +1237,9 @@ export default function (view) {
                     }
                     return;
                 case 'Enter':
-                    playbackManager.playPause(currentPlayer);
+                    if (e.target.tagName !== 'BUTTON') {
+                        playbackManager.playPause(currentPlayer);
+                    }
                     showOsd(btnPlayPause);
```

Comment:
We can also keep the OSD hidden. Some users are even against displaying the OSD on pause, but that's a question for UI designers.
```suggestion
                        showOsd(btnPlayPause);
                    }
```